packages/react-native-web/src/exports/ScrollView/index.js

```diff
@@ -189,10 +191,11 @@ const ScrollView = ((createReactClass({
         children={children}
         collapsable={false}
         ref={this._setInnerViewRef}
-        style={StyleSheet.compose(
+        style={[
```

Comment:
Every example of `StyleSheet.compose` throughout the repo had only two styles passed to it. Since this one now has three, I turned it into an array. If that's incorrect, feel free to let me know.